cli/cli_app.go

```diff
@@ -52,7 +52,7 @@ const (
 	optTerragruntStrictInclude                  = "terragrunt-strict-include"
 	optTerragruntParallelism                    = "terragrunt-parallelism"
 	optTerragruntCheck                          = "terragrunt-check"
-	optTerragruntHCLFmt                         = "terragrunt-hclfmt-file"
```

Comment:
I think it will be helpful to keep the backward compatibility of option names, to keep working existing HclFmt calls.

Also will be helpful to update existing documentation mentioning that multiple arguments are supported
 https://github.com/gruntwork-io/terragrunt/blob/master/docs/_docs/04_reference/cli-options.md#terragrunt-hclfmt-file

Reply:
I was thinking of a little abrupt change, by adding a check that fails with a little message when the old argument is passed -- I'm basing this on the fact that `terragrunt` is not stable yet -- is there any backward compatibility docs for terragrunt itself? 

Please let me know what you think? -- I'll also update the pre-commit repo

Reply:
I think we need to make switching between versions less complicated as possible, keeping same option names will make upgrades without any issue

Reply:
It should be good now